crates/bevy_ecs/src/event/collections.rs

```diff
@@ -264,6 +267,51 @@ impl<E: Event> Events<E> {
         self.events_a.start_event_count
     }
 
+    /// Trigger stored events that have not been triggered yet.
```

Comment:
I'd like to see a bit more explanation here; cross-links would be particularly valuable.